v2/pubsub-binary-to-bigtable/docs/PubSubAvroToBigtable/README.md

```diff
@@ -0,0 +1,159 @@
+# Pub/Sub Avro to Bigtable
+
+A Dataflow pipeline to stream [Apache Avro](https://avro.apache.org/) records
+from Pub/Sub to Bigtable. Any non-transient errors writing to the Bigtable table
+will be pushed to a Pub/Sub topic used as a dead-letter.
+
+## Requirements
+
+These are common requirements for all of the templates in this collection.
+
+*   Java 8
+*   Maven
+*   Docker
+
+## Getting Started
+
+### Building Template
+
+This is a Flex Template meaning that the pipeline code will be containerized and
+the container will be used to launch the Dataflow pipeline.
+
+### Compiling the pipeline
+
+Execute the following command from the directory containing the parent pom.xml
+(v2/):
```

Comment:
I think general suggestion is to execute at root, and `-pl v2/...`, can you make that adjustment?